src/dbseeder/dbseeder.py

```diff
@@ -153,6 +154,17 @@ def post_process(self, who):
             if c:
                 del c
 
+    def update(self, source, who):
+        db = self._get_db(who)
+
+        programs = self._parse_source_args(source)
+
+        for program in programs:
+            seederClass = factory.create(program)
+
+            seeder = seederClass(db)
+            seeder.update()
+
```

Comment:
Need to add `populateParamsTable` somewhere after the update to make sure that the `Params` table gets updated.